vhdbuilder/packer/install-dependencies.sh

```diff
@@ -248,6 +248,15 @@ if grep -q "fullgpu" <<< "$FEATURE_FLAGS" && grep -q "gpudaemon" <<< "$FEATURE_F
     systemctlEnableAndStart nvidia-device-plugin || exit 1
 fi
 
+if grep -q "sgxInstall" <<< "$FEATURE_FLAGS"; then
```

Comment:
no, I'm not saying we reuse feature_flags that we pass in manually.
We need a new argument in vhdbuilder/packer/vhd-image-builder-gen2.json where it call this install-dependencies.sh

Reply:
Talked offline, agreed that this will be fixed so no need to be part of feature flags, we can have it a s a separate flag